src/PanelContainer.ts

```diff
@@ -173,26 +173,36 @@ export class PanelContainer implements IDockContainerWithSize {
         }
     }
 
-    static createContextMenuContentCallback = (panelContainer: PanelContainer, contextMenuContainer: HTMLDivElement) => {
-        let btnNewBrowserWindow = document.createElement('div');
-        btnNewBrowserWindow.innerText = Localizer.getString('NewBrowserWindow');
-        contextMenuContainer.append(btnNewBrowserWindow);
+    static createContextMenuContentCallback = (panelContainer: PanelContainer): Node[] => {

```

Comment:
Is there a reason to mark this function as `static` and pass `this` as parameter, rather than using a class method ?

Reply:
yes, so you could override the contextmenu generation from outside

Reply:
I use it in my application to provide my own:

![image](https://github.com/node-projects/dock-spawn-ts/assets/364896/0309c3fa-3df0-4268-a2be-ab95344aed0b)

Reply:
We also could add a callback in the options, contextMenuProvider callback or smth similar...
Was the fastest way when I implemented it....
At some time in the future, we should switch to "popover API" for the ctx menu